engine/runtime/src/test/java/org/enso/interpreter/caches/ModuleCacheTest.java

```diff
@@ -78,10 +81,11 @@ public void testCompareWithWarning() throws Exception {
     var module = option.get();
     var ir = module.getIr().duplicate(true, true, true, true);
     var cm = new ModuleCache.CachedModule(ir, CompilationStage.AFTER_CODEGEN, module.getSource());
-    byte[] arr = module.getCache().serialize(ensoCtx, cm);
+    var mc = module.getCache().asSpi(ModuleCache.class);
+    byte[] arr = mc.serialize(ensoCtx, cm);
 
     var meta = new ModuleCache.Metadata("hash", "code", CompilationStage.AFTER_CODEGEN.toString());
-    var cachedIr = module.getCache().deserialize(ensoCtx, arr, meta, null);
+    var cachedIr = mc.deserialize(ensoCtx, ByteBuffer.wrap(arr), meta, null);
```

Comment:
Deserialization now works with `ByteBuffer`, not `byte[]` to allow _mmapped files_.